packages/agents/cartographer/poller/src/lib/operations/transfers.ts

```diff
@@ -252,6 +256,13 @@ export const updateBackoffs = async (): Promise<void> => {
     })
     .filter((x) => !!x) as { domain: string; checkpoint: number }[];
 
+  // update transfers with relayer fee bumps
+  // query subgraphs to avoid recalculating relayer fee based on increases
```

Comment:
Should we do the same for slippage updates too ?

Reply:
We do, above.

Reply:
https://github.com/connext/monorepo/pull/3572/files#diff-f23a559cc9ba71d6ad70ac0518924cac925aee6f3357b5482223ce1184ddb655R237

We have to handle this differently because the event does not emit the total relayer fee, just the increase.